migrations/migrations.amlg

```diff
@@ -271,5 +271,5 @@
 
 		(assign_to_entities updated_model_atributes_map)
 	)
-
-))
\ No newline at end of file
+"80.0.0"
```

Comment:
what's this for? this makes it an invalid assoc

Reply:
well invalid might be a strong word, it'll just have this key with a (null) value

Reply:
whoops, at a certain point i thought i needed this, was playing around